c++11/include/opentracing/util.h

```diff
@@ -0,0 +1,58 @@
+#ifndef OPENTRACING_UTIL_H
+#define OPENTRACING_UTIL_H
+
+#include <opentracing/version.h>
+#include <chrono>
+#include <opentracing/martinmoene_expected/expected.hpp>
```

Comment:
I am stuck in C++98-land, so I apologize if this is stupid: it seems weird to me to add a dependency like this to the *core* OpenTracing API. AFAICT, `expected` is becoming / becomes part of `std` in C++17, is that correct? Might it make more sense to just disallow exceptions in OT impls for the time being??

If the answer is some variant of "this is how people program in C++11", do not bother being verbose about it – I won't argue back, just wanted to ask the question.